pdf/creator/const.go

```diff
@@ -54,3 +59,32 @@ func (p positioning) isRelative() bool {
 func (p positioning) isAbsolute() bool {
 	return p == positionAbsolute
 }
+
+var (
+	// Default fonts used by all the components.
+	defaultFont     *model.PdfFont
```

Comment:
Would be better to do on a per-creator basis?
For example an application generating multiple PDFs in separate go-routines, could possibly go very wrong if defaultFont is altered.

Reply:
@gunnsth Yes, I thought about thread-safety problem. However, the components do not have access to the creator in any way. You can have an instance of a component without instantiating a creator.

If all the components were to be created via the creator interface, this would work. Something in the lines of:

```
c := creator.New()
paragraph := c.NewParagraph("text")
```

However, if you were to use the same creator component on multiple go-routines this would still not solve the problem.

Reply:
One common use we could think of is an API for generating reports.  An endpoint "/make_report" generates a PDF from input parameters.  In this case it makes a new Creator instance, creates the report and outputs the PDF.

I don't see a practical use case for multiple goroutines using the same Creator instance for generating PDFs, that is always going to be problematic.  Similar as having multiple threads writing to the same file at the same time (no synchronization).

Reply:
Yes, that would be a valid use case. But as I said, in order to have default fonts per creator, a creator instance must exist and the components must have access to it in order to access the default fonts. But as it is now, the components are completely separate from the creator.

Reply:
What about your suggestion to always go through the creator when creating components, e.g.
c.NewParagraph() etc.  ?  I think that could work, not sure if there are any disadvantages to that.  This signature is already used in a few components I believe (where context is required).  For example: `c.NewChapter()`

Any disadvantages to that?

Reply:
No, I think it makes sense for components to be instantiated through the creator object because they can only be used in the context of the creator. This will change the way the library is used, but I think it will provide a better interface overall. Let me know if you agree to proceed in this fashion.

Reply:
Great.  I think that's a good idea moving forward, let's do it.